vcpkg.json

```diff
@@ -1,16 +1,23 @@
 {
-    "name": "azure-sdk-for-cpp",
-    "version-string": "1.0.0",
-    "dependencies": [
-        {
-            "name": "curl"
-        },
-        {
-            "name": "libxml2",
-            "platform": "!windows & !uwp"
-        },
-        {
-            "name": "openssl"
-        }
-    ]
+  "name": "azure-sdk-for-cpp",
+  "version-string": "1.0.0",
+  "builtin-baseline": "f0aa678b7471497f1adedcc99f40e1599ad22f69",
+  "dependencies": [
+    {
+      "name": "curl"
+    },
+    {
+      "name": "libxml2",
+      "platform": "!windows & !uwp"
+    },
+    {
+      "name": "openssl"
+    }
+  ],
+  "overrides": [
+    {
+      "name": "openssl",
+      "version-string": "1.1.1n"
+    }
+  ]
```

Comment:
This file is **only** for building the SDk locally from source (using VCPKG)

I am curious about what's the story about people using VCPKG to install a service (letting VCPKG build the package)

Reply:
Yes, I know - but that's the file that our CI system and we use daily when we build, so it's the one that needs to be updated.

I am having a separate discussion about what we do w.r.t. the vcpkg.json files in each package directory (which I believe are functionally ignored today) and the vcpkg.json file we ship (there are 37 different vcpkg.json files in the tree :)).